src/components/Explore/ByLabelRepeater.tsx

```diff
@@ -44,6 +44,8 @@ export class ByLabelRepeater extends SceneObjectBase<ByLabelRepeaterState> {
       if (data.state.data) {
         this.performRepeat(data.state.data);
       }
+
+      return () => this._subs.unsubscribe();
```

Comment:
RTFM https://grafana.com/developers/scenes/advanced-data#use-data-in-a-custom-object-class. you don't need to return any subs added to this._subs!

> The subscription returned from sourceData.subscribeToState is added to this._subs. Because of this, you don't need to do any cleanup when the custom object is destroyed, as the library will take care of unsubscribing.

So this sounds like as long as we add everything to this._subs we are good to go